frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -128,3 +121,22 @@ export const InventoryItemLoading: React.FC = require('@console/shared/src/compo
   .InventoryItemLoading;
 
 export { useFlag } from '../utils/flags';
+
+/**
+ * A lazy component that creates a monaco editor
```

Comment:
the caller needs to render a `Suspense` before rendering the editor since its exported with `lazy`.
The component being lazy is vital information IMO @spadgett

Reply:
Is that something we can do in our wrapper so the caller doesn't need to?